README.md

```diff
@@ -62,6 +62,7 @@ All these endpoints accepts any combination of the following parameters:
 * `month`
 * `subquota_id`
 * `suspicions` (_boolean_, `1` parses to `True`, `0` to `False`)
+* `has_receipt` (_boolean_, `yes` parses to `True`, `no` to `False`)
```

Comment:
It's just about this line, right?

Reply:
Yes… but… may we keep the sabe _bool string_ documented for both suspicions and receipt? For suspicions we say `1` our `0`, for receipt you suggested `yes` or `no`…

Reply:
I forgot to ask that before comiting, but, since you are using 'yes' and 'no', why here it's 1 and 0?

Reply:
Addressing your question directly: the pint is _where_ we're using `1`/`0` and _where_ we're using `yes`/`no`. For the [dashboard (`admin.py`)](https://jarbas.serenatadeamor.org/dashboard/core/reimbursement) we use `yes`/`no`. For the [JSON API](https://jarbas.serenatadeamor.org/api/reimbursement/) we use `1`/`0`.

Why? Basically _bad design_, uninformed decisions in the past that _totally_ should be fixed — but I think this is an issue for a new PR ; )